src/Stratis.Bitcoin.IntegrationTests/Program.cs

```diff
@@ -11,6 +11,9 @@ internal class Program
     {
         public static void Main(string[] args)
         {
+            new NodeSyncTests().Pos_NodesCanConnect_AndSync_AndMineBlocks();
```

Comment:
I assume this is going to get removed when this PR is finished?

Reply:
yes yes lol I will remove it now